components/chat/ActiveWordsModal.tsx

```diff
@@ -5,44 +5,59 @@ import { useDispatch, useSelector } from 'react-redux';
 import { selectCurrentConversation } from '../../redux/chatSelectors';
 import Divider from '../common/Divider';
 import { useEffect } from 'react';
-import { useGetAllConversationsQuery } from './api';
+import { useGetAllConversationsQuery, useGetConversationQuery } from './api';
 import { isDataResponse } from '../../services';
 import { updateSelectedConversation } from '../../redux/chatSlice';
+import LoadingIndicator from '../common/feedback/LoadingIndicator';
 
 interface ActiveWordsModalProps {
   visible: boolean;
   setVisible: (visible: boolean) => void;
 }
 
 const ActiveWordsModal = ({ visible, setVisible }: ActiveWordsModalProps) => {
-  const { refetch } = useGetAllConversationsQuery();
-  const dispatch = useDispatch();
   const conversation = useSelector(selectCurrentConversation);
+  const dispatch = useDispatch();
+  const { data: latestConvoDetails, isLoading } = useGetConversationQuery(conversation?.id, {
```

Comment:
How would it be if this was isFetching like we did in other components?

Reply:
I knowingly did it that way, because I don't want the loading screen to show up each time